examples/connext_dds/instance_statistic/README.md

```diff
@@ -0,0 +1,46 @@
+# Example Code: Print QoS
+
+Example about getting instance statistic.
+
+## Concept
+
+RTI Connext DDS provides information about the instances that are currently
+being maintained by that DataReader or DataWriter.
+
+| Structure | Field Name | Description |
+| --------- | ---------- | ----------- |
+| DataReaderCacheStatus | alive_instance_count | The number of instances in the DataReader's queue with an instance state equal to DDS_ALIVE_INSTANCE_STATE. |
+| DataReaderCacheStatus | no_writers_instance_count | The number of instances in the DataReader's queue with an instance state equal to DDS_NOT_ALIVE_NO_WRITERS_INSTANCE_STATE. |
+| DataReaderCacheStatus | disposed_instance_count | The number of instances in the DataReader's queue with an instance state equal to DDS_NOT_ALIVE_DISPOSED_INSTANCE_STATE. |
+| DataWriterCacheStatus | alive_instance_count | The number of instances currently in the DataWriter's queue that have an instance_state equal to DDS_ALIVE_INSTANCE_STATE. |
+| DataWriterCacheStatus | disposed_instance_count | The number of instances currently in the DataWriter's queue that have an instance_state equal to DDS_NOT_ALIVE_DISPOSED_INSTANCE_STATE (due to, for example, being disposed via the FooDataWriter_dispose operation). |
+| DataWriterCacheStatus | unregistered_instance_count | The number of instances currently in the DataWriter's queue that the DataWriter has unregistered from via the FooDataWriter_unregister_instance operation. |
+
```

Comment:
Add a sentence saying that these statistics are also published by the monitoring libraries and can be viewed in RTI monitor